content/en/security/application_security/threats/add-user-info.md

```diff
@@ -679,6 +683,18 @@ track_custom_event(tracer, event_name, metadata)
 
 {{< /programming-lang-wrapper >}}
 
+## Disabling automatic user activity event tracking
+
+When ASM is enabled, recent Datadog Tracing Libraries will attempt to detect user activity events automatically.
+
+The events that can be automatically detected are:
+
+- `users.login.success`
+- `users.login.failure`
+- `users.signup`
+
+If you wish to disable the detection of these events, you should set the environment variable <code>DD_APPSEC_AUTOMATED_USER_EVENTS_TRACKING=disabled</code> (on the application hosting the Datadog Tracing Library, not on the Datadog Agent).
```

Comment:
In ruby we you can also disable using:

```ruby
Datadog.configure do |c|
  c.appsec.user_events.enabled = false
end
```

Should we mention how the different libraries can disable using a language block?